R/pkg_ref_cache_archive_release_date.R

```diff
@@ -0,0 +1,24 @@
+#' Cache a List of Archived Package Release Date from a Package Reference
+#'
+#' @inheritParams pkg_ref_cache
+#' @family package reference cache
+#'
+pkg_ref_cache.archive_release_date <- function(x, name, ...) {
+  UseMethod("pkg_ref_cache.archive_release_date")
+}
+
+pkg_ref_cache.archive_release_date.pkg_remote <- function(x, name, ...) {
```

Comment:
I think that this url (`"%s/src/contrib/Archive/%s"`) is CRAN-specific. I got an error when trying to run with a Bioconductor package. Let's leave it as is, but leave the Bioconductor version undefined and only run this for CRAN remotes. 

```suggestion
pkg_ref_cache.archive_release_date.pkg_cran_remote <- function(x, name, ...) {
```